package/gluon-mesh-vpn-wireguard/files/lib/gluon/upgrade/404-wireguard-generate-key

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+secretfile=/lib/gluon/mesh-vpn/wireguard
+if ! wg pubkey < $secretfile >/dev/null 2>&1
```

Comment:
Wouldn't using "||"-Shell logic make this more elegant than an if-clause?

Reply:
`wg pubkey < $secretfile >/dev/null 2>&1 || wg genkey > $secretfile` ?
Isn't this the same?

Reply:
matter of taste I guess. I like both representations and think the slightly more verbose one is more readable.